lib/utils/visFile.cpp

```diff
@@ -106,7 +107,12 @@ void visFileBundle::add_file(time_ctype first_time) {
 
     // Construct the name of the new file
     uint32_t time_since_start = (uint32_t)(first_time.ctime - acq_start_time);
-    std::string file_name = fmt::format(fmt("{:08d}_{:04d}"), time_since_start, freq_chunk);
+    std::string file_name;
```

Comment:
You might consider just leaving the chunk in the name here for uniformity. It is a bit daft as we don't use it, but maybe better than the "if hfb" special case.

Reply:
Particularly as your test code above still adds the chunk digits.